src/main/java/io/confluent/connect/jdbc/sink/DbStructure.java

```diff
@@ -112,7 +111,7 @@ void create(
       final FieldsMetadata fieldsMetadata
   ) throws SQLException {
     if (!config.autoCreate) {
-      throw new ConnectException(
+      throw new TableAlterOrCreateException(
```

Comment:
I think we have left out that these methods throw `TableAlterOrCreateException` and the caller does not handle these aswell. This has bubbled up in an incident recently. https://confluent.pagerduty.com/incidents/P1M3JE2